.github/workflows/test.yml

```diff
@@ -70,6 +70,8 @@ jobs:
         with:
           command: test
           args: --all-features
+        env:
+          PHYLUM_API_KEY: ${{ secrets.PHYLUM_TOKEN }}
```

Comment:
This token is for the _production_ system. The "invalid refresh token" errors in the tests now might be fixed if a different token, for the _staging_ environment is used instead. @furi0us333 added the `PHYLUM_TOKEN` as an organizational secret...and might be the right person to add the token for staging. It appears a relevant account already exists in the password manager.

NOTE: If the organizational secret names are changed (e.g., to differentiate between a PRODUCTION and STAGING token), then all the repos that make use of the current `PHYLUM_TOKEN` org secret will also need to be updated.